packages/api-nest/test/helpers/create-e2e-app.ts

```
@@ -0,0 +1,159 @@
+ *   afterAll(async () => {
+ *     await app.close()
+ *   })
+ *
+ *   it('should work', async () => {
+ *     const response = await request(app.getHttpServer())
+ *       .get('/api/v2/health')
+ *       .expect(200)
+ *   })
+ * })
+ * ```
+ */
```

ESLint

[error] Missing JSDoc return type.

(valid-jsdoc)